students/jvirtue/lesson06/test_mailroom_ut.py

```diff
@@ -0,0 +1,48 @@
+#Lesson 6 Assignment 1
+#Mailroom Part 4 Assignment
+#Jason Virtue 03/02/2019
+#UW Self Paced Python Course
+
+"""
+Unit test code for the mailroom_ut.py
+"""
+
+#Import Mailroom package
+import pytest
+import mailroom_ut as m
+import os
+import datetime
+
+#Test main menu is INT
+test_donor_db = {'Fred Flintstone': [100,200],'Wilma Flintstone': [300],'Bamm-Bamm Rubble': [50,30,40],'Barney Rubble' : [75],'Pebbles Flintstone': [500]}
+
+#List out Valid donors from List function
+def test_donor_list():
+    test_donors = ['Fred Flintstone','Wilma Flintstone','Bamm-Bamm Rubble','Barney Rubble','Pebbles Flintstone']
```

Comment:
common pattern is to use `expected` and `actual` for variable names (for clarity)